### DISCUSSION TITLE
automatically read email recipient from printed document for email send action

### DISCUSSION THREAD

**Discussion by ThinkEV** [clawsoftware/clawPDF#78] [Ideas]
Hello,

I want to write my business letters in openoffice writer and instead of printing and mailing it, I want to use clawPDF and print it to the email action.
I am writing therecipients email address into my document. It shall be extracted from the document, using the iText library, for example.

It should be detected by user configurable PCRE that picks the adress.
I am using something like this in my document:
`"per E-Mail: <address>\w"` (maybe multiple times for multiple recipients)

And in addition to that, I would like to set custom variables (my reference, your reference) that should be usable in the email subject and message body.

sincerely

**Comment by clawsoftware** [comment]
@ThinkEV This is a good idea. I will include this in the upcoming versions.

**Reply by ThinkEV** [reply]
Maybe you can add a new settings tab "PCRE" where you can define 1..n PCRE search strings that populate $PCRE1 .. $PCREn that can be used as token.

**Comment by clawsoftware** [comment]
@ThinkEV I have implemented a simple recipient recognition. In future versions I may extend this.

**Reply by ThinkEV** [reply]
@clawsoftware Please supply some documentation on how to use it.
